Returning iterable of string instead of bytes in HEAD response

**Describe the bug**
uWSGI is expecting an iterable of bytes instead of (empty) string which is returned in case of HEAD request. This results in logging `[ERROR] Unhandled object from iterator: ''` because  [_The iterator returned something that we were not able to handle._](https://github.com/unbit/uwsgi/blob/d58a832c81c2c96ae0f6e72614e1cc47f4b5d332/plugins/python/wsgi_subhandler.c#L355)

**To Reproduce**
1. Create HEAD request
2. Check uwsgi logs for mentioned error log

--- a/gluon/http.py
+++ b/gluon/http.py
@@ -122,7 +122,7 @@ class HTTP(Exception):
                 rheaders.append((k, str(v)))
         responder(status, rheaders)
         if env.get('request_method', '') == 'HEAD':
-            return ['']
+            return [to_bytes('')]
         elif isinstance(body, (str, bytes, bytearray)):
             if isinstance(body, unicodeT):
                 body = to_bytes(body)

